pnpm-lock.yaml

```diff
@@ -405,7 +405,123 @@ importers:
         version: 5.8.3
       vitest:
         specifier: 'catalog:'
-        version: 3.2.4(@types/debug@4.1.12)(@types/node@20.17.30)(tsx@4.19.3)(yaml@2.7.1)
+        version: 3.2.4(@types/debug@4.1.12)(@types/node@22.16.5)(jiti@2.5.0)(lightningcss@1.30.1)(tsx@4.20.3)(yaml@2.7.1)
+
+  packages/debug-ui-vibe:
```

Comment:
was that supposed to be reverted?

Reply:
blah I dunno, I just did pnpm install is all. I'll rebase main and see what I get.